.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: kotlin
 jdk:
-- oraclejdk8
+- oraclejdk11
```

Comment:
I'd recommend using the openjdk here so it's consistent with what we use for deployment. This is licensing related.